packages/private/legacy-editor-to-editor/src/splishToEdtr/types.ts

```diff
@@ -59,14 +59,26 @@ export function isContentCell(cell: Cell): cell is ContentCell {
 
 type SplishPlugin = { name: Plugin | 'code'; version?: string }
 
-export type Edtr = RowsPlugin | LayoutPlugin | OtherPlugin
+export type Edtr = RowsPlugin | LayoutPlugin | SolutionStepsPlugin | OtherPlugin
 
 export type RowsPlugin = { plugin: 'rows'; state: Edtr[] }
 export type LayoutPlugin = {
   plugin: 'layout'
   state: { child: Edtr; width: number }[]
 }
 
+export type SolutionStepsPlugin = {
```

Comment:
Do we need that explicitly? Should be fine as part of `OtherPlugin` I suppose?

Reply:
accepted.